### ISSUE TITLE
[Bug]: googlechrome has been outdated for months

### ISSUE THREAD

**Issue by sqliuchang** [ScoopInstaller/Extras#10172]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

googlechrome

### Expected/Current Behaviour

The package googlechrome has been outdated for months.
And there was a pull request to solve it 2 weeks ago, but it stiil hasn't been reviewed and merged.
issue: [Updates are blocked by ryzen-controller](https://github.com/ScoopInstaller/UpdateTracker/issues/6)
pr: [Fix ryzen-controller check](https://github.com/ScoopInstaller/UpdateTracker/pull/8)

### Steps to Reproduce

```console
https://github.com/ScoopInstaller/Extras/blob/master/bucket/googlechrome.json
"version": "107.0.5304.107" vs "currentversion": "108.0.5359.125"
```

### Possible Solution

There has been a pr mentioned before.

### Scoop and Buckets Version

```console
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'extras' bucket:
d931e5fb6 (HEAD -> master, origin/master, origin/HEAD) telegram-downloader: Update to version 0.6.0

'main' bucket:
fd8ea2c7a (HEAD -> master, origin/master, origin/HEAD) minio: Update to version 2023-01-02T09-40-09Z

'nonportable' bucket:
7ced35e (HEAD -> master, origin/master, origin/HEAD) spotx-np: Update to version 1.7-20230101-3a2b64

'versions' bucket:
632de1f72 (HEAD -> master, origin/master, origin/HEAD) r-patched: Update to version 4.2.2-r83548
```

### Scoop Config

```console
last_update       scoop_repo                              scoop_branch
-----------       ----------                              ------------
2023/1/3 20:45:12 https://github.com/ScoopInstaller/Scoop master
```

### PowerShell Version

```console
Name                           Value
----                           -----
PSVersion                      7.3.1
PSEdition                      Core
GitCommitId                    7.3.1
OS                             Microsoft Windows 10.0.19044
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by Lutra-Fs** [comment]
I’ve merged the workaround now. Chrome series should be upfated in the next check

**Comment by Lutra-Fs** [comment]
For the solution I need time to review it.

**Comment by sqliuchang** [comment]
> I’ve merged the workaround now. Chrome series should be upfated in the next check

Thank you!

**Comment by Lutra-Fs** [comment]
> Chrome is outdated for 2 major releases
> 
> 
> 
> 109 vs 111
> 
> 
> 
> https://chromereleases.googleblog.com/2023/03/stable-channel-update-for-desktop.html
> 
> 
> 
> ```
> 
> C:\>scoop search chrome
> 
> Results from local buckets...
> 
> 
> 
> Name               Version                 Source Binaries
> 
> ----               -------                 ------ --------
> 
> chromium           109.0.5414.120-r1070088 extras chrome.exe
> 
> dartium            1.24.2                  extras chrome.exe
> 
> googlechrome       109.0.5414.129          extras
> 
> ungoogled-chromium 110.0.5481.177-1.1      extras chrome.exe
> 
> chromedriver       111.0.5563.64           main
> 
> ```

You should raise a new issue. I have already update the upstream to drop windows 7 support since 109 is the last version supported.
